src/pybind/mgr/prometheus/module.py

```diff
@@ -400,6 +424,41 @@ def get_quorum_status(self):
                 'mon.{}'.format(id_),
             ))
 
+    def get_mgr_status(self):
+        mgr_map = self.get('mgr_map')
+        servers = self.get_service_list()
+
+        active = mgr_map['active_name']
+        standbys = [s.get('name') for s in mgr_map['standbys']]
+
+        all_mgrs = list([active])
+        all_mgrs.extend(standbys)
+
+        all_modules = {module.get('name'):module.get('can_run') for module in mgr_map['available_modules']}
+        
+        active_modules = []
+        active_modules.extend(mgr_map['always_on_modules'])
+        active_modules.extend(mgr_map['modules'])
+
+        for mgr in all_mgrs:
+            _state = 1 if mgr == active else 0
+            host_version = servers.get((mgr, 'mgr'), ('', ''))
+            self.metrics['mgr_metadata'].set(1, (
+                'mgr.{}'.format(mgr), host_version[0],
+                host_version[1]
+            ))
+            self.metrics['mgr_status'].set(_state, (
+                'mgr.{}'.format(mgr), 
+            ))
+
+        for mod_name in all_modules.keys():
+            _state = 1 if mod_name in active_modules else 0
+            _runnable = 'Yes' if all_modules[mod_name] else 'No'
```

Comment:
I am not too set on it but could we use `1` and `0` here as well? Mostly for the sake of consistency. The strings are also more difficult to guess right when matching, etc.

Reply:
Agreed, please use either `1` and `0` or `True` and `False`.

Reply:
Style changes applied in latest commit